app/frontend/walletApp.js

```diff
@@ -1,3 +1,4 @@
+// import 'preact/debug'
```

Comment:
is this left-over?

Reply:
Nope, this is a purposefully left comment, so If a developer needs to use preact dev tools, he just uncomments this.

Reply:
In that case I would add an additional comment after this

Reply:
Fair point :+1: